lib/rails/graphql/type/interface.rb

```diff
@@ -46,14 +46,14 @@ def validate!(object = nil)
 
```

Comment:
We won't need this function anymore since the fields are now being copied.
I also think we shouldn't validate interfaces because there might be broken types,
So just:
```ruby
def validate(*)
  # Don't validate interfaces since the fields are copied and
  # the interface might have broken field types
end
```